Makefile

```diff
@@ -37,3 +44,13 @@ validate_translation_files:  ## Run basic validation to ensure files are compila
 	@echo '-----------------------------------------'
 	@echo 'Congratulations! Translation files are valid.'
 	@echo '-----------------------------------------'
+
+sync_requirements:  ## install sync.txt requirements
+	pip install -q -r requirements/sync.txt
```

Comment:
I think we should keep a single `translations.in` requirements file. Too many requirements files are a pain to manage.

Reply:
done